src/DependencyInjection/Configuration.php

```diff
@@ -565,6 +566,20 @@ private function addAdapterSection(ArrayNodeDefinition $node): void
             ->end();
     }
 
+    private function addMessengerSection(ArrayNodeDefinition $node): void
+    {
+        $node
+            ->children()
+                ->arrayNode('messenger')
+                    ->canBeEnabled()
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('bus')->isRequired()->defaultValue('messenger.default_bus')->end()
+                    ->end()
+                ->end()
```

Comment:
IMO we should allow configure bus for each message (which can be more then one in the future):
```suggestion
                ->arrayNode('messenger_buses')
                    ->canBeEnabled()
                    ->addDefaultsIfNotSet()
                    ->children()
                        ->scalarNode('generate_thumbnails')->isRequired()->defaultValue('messenger.default_bus')->end()
                    ->end()
                ->end()
```

Reply:
Changed to reflect the bus is only for generate thumbnails.